c2corg_ui/templates/waypoint/index.html

```diff
@@ -1,11 +1,34 @@
-<%inherit file="../base.html"/>
+<%inherit file="../base.html"/>\
 
 <%block name="pagetitle">
 Waypoints 
-</%block>
+</%block>\
+
+<%block name="moduleConstantsValues">
+% if len(waypoints):
+module.value('mapFeatureCollection', {
```

Comment:
Right. I haven't done it yet because I thought we should wait for the integration of the other doc types.